### DISCUSSION TITLE
Backport error: How to check what went wrong?

### DISCUSSION THREAD

**Discussion by defung** [Mergifyio/mergify#2967] [Q&A]
I'm trying to test the backport command:

```
@Mergifyio backport mergify-test-main
```

I got a thumbs up emoji once I added that comment into my PR, and a few seconds later I got a message saying that it is currently waiting for the PR to be merged.

Once I merged the PR however, I got the following message:

**Command `backport mergify-test-main`: failure**
> **No backport have been created**
> * Backport to branch `mergify-test-main` failed

No PR was created.

Can someone let me know how I can troubleshoot this? Where can I go to check why this failed?

**Comment by jd** [comment]
It might be an error we didn't catch correctly on our side. Can you share the PR URL?

**Reply by defung** [reply]
I can't, as we are using this in an internal enterprise github server.

**Reply by jd** [reply]
Alright, asking your system administrator to take a look at the log might be worth it then.
